### PR TITLE
Update `get-started` with resolutions of questions from demo

### PR DESCRIPTION
**Pull request by falexwolf**
This is meant to resolve the two below issues:

- https://github.com/laminlabs/laminr/issues/103
- https://github.com/laminlabs/laminr/issues/108

### CODE REVIEW

**Comment by falexwolf** [comment]
Why does this fail? I just copied from Robrecht's notebook:

image

https://lamin.ai/laminlabs/lamindata/transform/y0MIazBuljv70002

**Comment by falexwolf** [comment]
In the Python API, we allow to `artifact.open()` tiledbsoma stores.

It'd be great if we could replace the call of [open_soma](https://chanzuckerberg.github.io/cellxgene-census/r/reference/open_soma.html)

```R
census <- open_soma()
```

with this:

```
census <- artifact.open()
```

This is how it looks in Python: https://docs.lamin.ai/cellxgene#slice-a-concatenated-array

To make that happen, it'll only be a few lines of changes to the census package:

https://github.com/chanzuckerberg/cellxgene-census/blob/f32c2748e9664cc196bb1758acb22f5a460fe188/api/r/cellxgene.census/R/open.R#L24C1-L38C2

In particular, replacing the s3 path (uri in the code there) via `artifact$path` so that it works for any tiledbsoma store.

File: vignettes/laminr.Rmd [hunk 1]
```diff
@@ -115,40 +115,66 @@ This artifact contains an [`AnnData`](https://anndata.readthedocs.io) object.
...
-Seurat::DotPlot(seurat, features = unique(markers$gene)) +
+Seurat::DotPlot(seurat_obj, features = unique(markers$gene)) +
   ggplot2::theme(axis.text.x = ggplot2::element_text(angle = 90, vjust = 0.5))
 ```
 
+# Slice the tiledbsoma array store
+
+Alternatively to querying individual datasets, you can also slice the `.tiledbsoma` array store, which stores Census, a concatenated version of most datasets in CELLxGENE.
+
+```{r create-seurat}
```
Comment (lazappi) [inline vignettes/laminr.Rmd:151]: There is a duplicate chunk label which is causing the rendering to fail

````suggestion
```{r slice-tiledb}
````

**Comment by falexwolf** [comment]
> Does this fit in this vignette? It doesn't seem like part of the introductory workflow and doesn't use {laminr}. Could we just point someone to the documentation for the {cellxgene.census} package if they want to know how to do this?

I agree it doesn't generally fit into this vignette, but right now the main audience for this is the person who asked the question; so it should be featured here on this page.

Once the below issue is addressed it will fit well, though:

- https://github.com/laminlabs/laminr/issues/114

So: Let's please keep it here for now but not for long in the present form. The section will be replaced with the `$open()` call in the upcoming weeks, which is fundamentally different from `$load()` and `$cache()` and hence merits to be shown in `get-started`.

Even without all this, it illustrates that you can alternatively get a Seurat object by slicing a big array store and then save the result into lamindb.

**Comment by falexwolf** [comment]
Can you please help to get this to pass, @lazappi? It's going to take you much less time than I. I'm simply not familiar enough.

**Comment by lazappi** [comment]
I adjusted the chunk a bit but there are some bigger issues:

1. The **{cellxgene.census}** package is currently not working (see https://github.com/chanzuckerberg/cellxgene-census/issues/1261). I haven't been able to run the code manually even with the workarounds listed here. (The Python package is having similar issues https://github.com/chanzuckerberg/cellxgene-census/issues/1313)
2. **{cellxgene.census}** is not available from CRAN/Bioconductor and only works on Unix. This means it is difficult for people to install and we can't really add it as a dependency so the CI will fail without some workarounds and CRAN probably won't allow it.

I think the best we could do at the moment is to have the code there but not evaluated, I'm not sure how useful that is if people can't install/use the package anyway though. There is a Bioconductor [**{cellxgenedp}**](https://www.bioconductor.org/packages/release/bioc/html/cellxgenedp.html) package that could maybe be used instead but I'm not sure if it does what you are looking for.

**Comment by falexwolf** [comment]
> I think the best we could do at the moment is to have the code there but not evaluated,

I agree.

> I'm not sure how useful that is if people can't install/use the package anyway though

It's useful to illustrate the point of streaming a big array to get a Seurat object vs. caching a small array locally. This is important to convey to the current target audience, even if the caveats that you're listing make clear that this is currently problematic. I'm expecting these issues will get resolved by CZI though.

If it's installable on UNIX that's probably good enough. My understanding is they all work on Linux servers in the cloud.

**Review by rcannood — APPROVED** [review]
LGTM

**Pull request merged**